src/transformers/utils/quantization_config.py

```diff
@@ -1090,6 +1090,8 @@ class CompressedTensorsConfig(QuantizationConfigMixin):
             configuration for sparsity compression
         quant_method (`str`, *optional*, defaults to `"compressed-tensors"`):
             do not override, should be compressed-tensors
+        run_compressed (`bool`, *optional*, defaults to `True`): compress weights (usually linear) modules
+            as compressed weights, using `format` method
```

Comment:
not sure to understand what it `format` method

Reply:
Format is the key to quantization confit, it tells what quant format to compress to - ex packed quantized (pack in int4,8 into int32 as contiguous array)